addons/l10n_hr/__manifest__.py

```diff
@@ -19,6 +19,7 @@
   - Goran Kliska, Exocdica d.o.o - original module for v6.1  
     (cla/ex : Decodio-applications d.o.o., ex Slobodni programi d.o.o.)
   - Davor Bojkić, Daj mi 5 - port to odoo 16, update tax schema to 2022
+  - Alberto Poljak, e-Sustavi d.o.o - update port for Odoo 16
```

Comment:
OK this is cool, but, you signed ILA as individual... (individual licence agreement )
why don't you have a company CLA signed?